roles/preflight-checks/tasks/check_items.yml

```diff
@@ -69,4 +69,10 @@
 - include: v7k.yml
   when: v7k.enabled
   tags: ['precheck_v7k']
-  run_once: true
\ No newline at end of file
+  run_once: true
+
+- name: Make sure barbican is enabled when cinder encryption is enabled
```

Comment:
So we don't support fixed_key, right? Even not for CI?
If that, we can remove all fixed_key related code.